client/tests/integration/extra_functional/unstable_network.rs

```diff
@@ -96,23 +97,22 @@ fn unstable_network(
             quantity,
             AssetId::new(asset_definition_id.clone(), account_id.clone()),
         );
-        iroha.submit(mint_asset).expect("Failed to create asset.");
+        iroha
+            .submit_blocking(mint_asset)
```

Comment:
Should we replace all uses of `submit` with `submit_blocking` and get rid of polling? Or does it matter less for non-extra-functional tests?

Reply:
I think we should by default opt to `submit_blocking` and use `submit` in places where it's really required.